src/components/Checkbox/index.test.tsx

```diff
@@ -15,57 +15,70 @@ describe('Checkbox component', () => {
     userEvent.click(input);
     expect(input).not.toBeChecked();
     expect(screen.getByRole('checkbox')).toHaveAttribute('disabled');
-    expect(container.querySelector('.fdsCheckable')).toHaveClass(
-      'fdsCheckable--disabled'
+    expect(container.querySelector('.fdsCheckableCheckbox')).toHaveClass(
+      'fdsCheckableCheckbox--disabled'
```

Comment:
I would either checked for some disabled attribute, or just remove this test. It would be one thing if were were testing different themes, but because we're not, I would just get rid of this.